ci/container_index/lib/checks/schema_validation.py

```diff
@@ -135,12 +135,12 @@ def _perform_validation(self):
         if (not isinstance(
                 self.validation_data.get(FieldKeys.PREBUILD_SCRIPT), str)):
             self._invalidate("prebuild-script must be a valid string.")
+        elif len(self.validation_data.get(FieldKeys.PREBUILD_SCRIPT)) <= 0:
```

Comment:
how come length be of `<  0` ?  `== 0` is just fine.

Reply:
Sorted